pandas/core/internals/managers.py

```diff
@@ -1610,6 +1610,8 @@ def get_slice(self, slobj: slice, axis: int = 0) -> SingleBlockManager:
 
         blk = self._block
         array = blk._slice(slobj)
+        if array.ndim > blk.values.ndim:
+            raise ValueError
```

Comment:
message?

Reply:
could add something in to be on the safe side, but this is only reached when called from Series.get_value, where it is caught and handled